app/assets/stylesheets/blacklight.scss

```diff
@@ -66,3 +66,37 @@
     }
   }
 }
+
+dt.blacklight-full_text_tesimv {
+  @extend .h4;
+  display: inline-block;
+  float: none;
+  text-align: initial;
+  width: auto;
+
+  &[data-toggle="collapse"]:after {
+    /* symbol for "opening" panels */
+    font-family: 'Glyphicons Halflings';  /* essential for enabling glyphicon */
+    content: "\e114";    /* adjust as needed, taken from bootstrap.css */
+    float: right;        /* adjust as needed */
+    color: grey;         /* adjust as needed */
```

Comment:
Color `grey` should be written in hexadecimal form as `#808080`<br>Color literals like `grey` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Use `//` comments everywhere